### PR TITLE
doc: remove reference to port file

### LINKED ISSUE #2137: CUG: remote control section needs updating
e.g. it refers to the obsolete port file directory.

### PR DESCRIPTION
**Pull request by matthewrmshin**
Should be sufficient to close #2137.

### CODE REVIEW

File: doc/cug.tex [hunk 1]
```diff
@@ -6203,38 +6203,33 @@ \subsection{Remote Control - Passphrases and Network Ports}
...
-account, so it is recommended that this only be used to interact with
-your own suites running on other hosts. We plan to implement
-finer-grained authentication in the future to allow suite owners to
-grant read-only access to others.
+Possession of a suite passphrase gives full control over the suite, and ssh
+access to the contact file also implies full access to the suite host account,
+so it is recommended that this only be used to interact with suites running on
+accounts that you already have full access. We plan to implement finer-grained
+authentication in the future to allow suite owners to grant read-only access to
+others.
```
Comment (hjoliver) [inline doc/cug.tex]: The last sentence is outdated - see site.rc `[authentication]public = full-read` (etc.)

Comment (hjoliver) [inline doc/cug.tex]: (but otherwise good, thanks).

Comment (matthewrmshin) [inline doc/cug.tex]: Sentence removed.

**Comment by matthewrmshin** [comment]
Doc updated. Branch squashed and re-based.

**Comment by hjoliver** [comment]
Simple doc correction - one review will do.

**Pull request merged**